src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -7,6 +7,9 @@
 #ifndef LINE_EXTRACTOR_IGVC_DBSCAN_H
 #define LINE_EXTRACTOR_IGVC_DBSCAN_H
 
+// run sequentially when point cloud size is less than SEQUENTIAL_CUT_OFF
+#define SEQUENTIAL_CUT_OFF 1000
```

Comment:
I replied to your comment on the previous review, but including it in this review for ease of reading:

"Hmm, I see what you mean, but I don't think it'd hurt to make it a class member, at the very least it might be a bit more clear that this is part of the class itself."

Reply:
done. just to clarify, do we want to make this a variable that we can configure from the launch file?

Reply:
From your previous comments, it doesn't seem like this is something that we should need to change (if I understand correctly)? In this case it's more about making it clear that it's part of the class and can be changed if need be, so I don't see a need to make it a rosparam as well at this point.